### PR TITLE
add parameters for bamford buzzard

### PR DESCRIPTION
**Pull request by SamaraMedina**
Please merge my species?

### CODE REVIEW

File: params/bamford_buzzard.R [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 3.2
+    
+# e.g. "anas_krystallinus"    
+species.name <- "bamford_buzzard"
```
Comment (ns-rse) [inline params/bamford_buzzard.R]: Hi @SamaraMedina we've had a few bamford buzzards, would you mind changing the `species.name` please?

**Review by ns-rse — CHANGES_REQUESTED** [review]
Thanks @SamaraMedina would you be able to rename the `species.name` and the file please.

Thanks @ns-rse

File: params/bamford_buzzard.R (hunk 1 above)
Comment (SamaraMedina) [reply params/bamford_buzzard.R]: Change done, thanks.

**Review by ns-rse — APPROVED** [review]
Thanks for the contribution @SamaraMedina looks good to merge :+1:

**Review by bobturneruk — APPROVED** [review]
That looks really good!

**Pull request merged**